eclipse-collections-code-generator/src/main/resources/api/block/function/primitiveComparator.stg

```diff
@@ -25,7 +25,7 @@ import java.io.Serializable;
  *
  */
 @FunctionalInterface
-public interface <name>Comparator\<T>
+public interface <name>Comparator
```

Comment:
Interesting, I guess we missed the unused generics on the initial commit. Good Catch.